torchvision/datasets/kinetics.py

```diff
@@ -20,44 +34,103 @@ class Kinetics400(VisionDataset):
     Note that we drop clips which do not have exactly ``frames_per_clip`` elements, so not all
     frames in a video might be present.
 
-    Internally, it uses a VideoClips object to handle clip creation.
-
     Args:
-        root (string): Root directory of the Kinetics-400 Dataset. Should be structured as follows:
-
+        root (string): Root directory of the Kinetics Dataset.
+            Directory should be structured as follows:
             .. code::
 
                 root/
-                ├── class1
-                │   ├── clip1.avi
-                │   ├── clip2.avi
-                │   └── ...
-                └── class2
-                    ├── clipx.avi
-                    └── ...
-
+                ├── split
+                │   ├──  class1
+                │   │   ├──  clip1.mp4
+                │   │   ├──  clip2.mp4
+                │   │   ├──  clip3.mp4
+                │   │   ├──  ...
+                │   ├──  class2
+                │   │   ├──   clipx.mp4
+                │   │    └── ...
+            Split is appended using the split argument.
+        num_classes (int): select between Kinetics-400, Kinetics-600, and Kinetics-700
+        split (str): split of the dataset to consider; currently supports ["train", "val"]
+        frame_rate (float): If not None, interpolate different frame rate for each clip.
         frames_per_clip (int): number of frames in a clip
         step_between_clips (int): number of frames between each clip
+        annotation_path (str): path to official Kinetics annotation file.
```

Comment:
I don't think we need this as parameter. What would be the use case?

Reply:
if someone decided to use their own additional annotations (e.g. for action detection) or different version of the annotations (legacy ones with old videos)

Reply:
actually, yeah, makes sense to remove this. Will do